compliance/index/v4/node.go

```diff
@@ -33,6 +36,16 @@ var (
 	log         = logging.LoggerForModule()
 )
 
+// ClairCore is using zlog and needs separate configuration, as it's logging on debug level by default.
+func configureClairCoreLogging() {
+	l := zerolog.New(os.Stderr)
+	l = l.Level(zerolog.InfoLevel)
+	if logging.GetGlobalLogLevel().CapitalString() == "DEBUG" {
+		l = l.Level(zerolog.DebugLevel)
+	}
```

Comment:
I think we should use the same level as `logging.GetGlobalLogLevel()` instead of forcing it to be either info or debug